frontend/app/src/screens/diff/data-diff-node.tsx

```diff
@@ -135,46 +120,14 @@ export type tDataDiffNodeProps = {
   branch?: string;
 };
 
-// Branch from QSP = branchName
-// Multiple branches = branches array
-// Related branch for the node update = branch
-export const getNodeClassName = (
-  branches: string[],
-  branch: string | undefined,
-  branchOnly?: string | null | undefined
-) => {
-  // Do not display a color if the node is related to mulitple branches or if we are on the branch details diff
-  if (branches?.length > 1 || branchOnly === "true" || !branchOnly) {
-    return "bg-custom-white";
-  }
-
-  if (branches?.length === 1) {
-    return branches[0] === "main" ? "bg-custom-blue-10" : "bg-green-200";
-  }
-
-  return branch === "main" ? "bg-custom-blue-10" : "bg-green-200";
-};
-
 export const DataDiffNode = (props: tDataDiffNodeProps) => {
```

Comment:
<img width="899" alt="image" src="https://github.com/user-attachments/assets/c2a5a0ce-73ea-4dd4-be4c-fa83cf73cd80">
 There is no visual indication to understand where comes the data from the green side and the blue side